geonode/management_commands_http/admin.py

```diff
@@ -80,8 +68,35 @@ def celery_state(self, instance):
     def celery_traceback(self, instance):
         return get_celery_task_meta(instance).get("traceback")
 
-    def has_add_permission(self, request, obj=None):
+    def has_change_permission(self, request, obj=None):
         return False
 
     def has_delete_permission(self, request, obj=None):
         return False
```

Comment:
The idea here, is to do not allow the deletion of already executed jobs (at least).
So if one admin wants to do an action it will be registered in the list and visible to others admin, it helps in the following ways:
* No one can execute something critical, then delete the job and pretends that nothing happened.
* If an admin account is hacked and used to run a job, it will be registered.
* It provides us a reliable list of executed jobs, that can be inspected.
* Not allowing the deletion of jobs increases the security of the system.

To handle the case of wrongly creating a job, I think I can change the `has_delete_permission` to verify the `job.status` and only allows the deletion of jobs with `Created` status. (Deletion not allowed for `Queued`, `Started` and `Finished` jobs).

What do you think about it?